src/System.IO.Pipelines/ref/System.IO.Pipelines.cs

```diff
@@ -29,7 +29,7 @@ public sealed partial class Pipe
     }
```

Comment:
Did not found anything helful to determine nullability of IDuplexPipe.Input, IDuplexPipe.Output on row 19, most likely they are not null

Reply:
Yes, they should be non-nullable.